app/views/shared/_flash.html.haml

```diff
@@ -18,7 +18,7 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 #flashes_container
-  - if !content_for?(:error_messages)
+  - if @rmid_server_down
```

Comment:
This seems like a very niche change. The only change is removing the `= content_for(:error_messages)` inside of the flash. @jwiel86 can you send me a slack message or just reply here why you needed to remove it for this one case?

Reply:
Approving this comment for now. Discussed with @jwiel86 and we decided that it would seem to work removing the `if-else` altogether, but to be safe for now we should leave it like this and revisit it later.